androidApp/src/main/java/com/radityalabs/qualitymatter/androidApp/MainActivity.kt

```diff
@@ -4,6 +4,11 @@ import androidx.appcompat.app.AppCompatActivity
 import android.os.Bundle
 import com.radityalabs.qualitymatter.shared.Greeting
 import android.widget.TextView
+import androidx.compose.material.MaterialTheme
+import androidx.compose.material.Surface
+import androidx.compose.material.Text
+import androidx.compose.ui.platform.setContent
+import com.radityalabs.qualitymatter.androidApp.ui.AppTheme
 
 fun greet(): String {
```

Comment:
this is mean to widget?

either we put on widget dir or inlining on the apptheme body itself. wdyt?

Reply:
Did you mean make them as @Composable?

Reply:
Something like this https://github.com/radityagumay/QualityMatters/pull/1/commits/c97990ef8c0f2fad4eee4059d6b843a1c703f5b5

Reply:
@wasinpp exactly!